src/ec/packages/ec-component-site-header-core/site-header-core.story.js

```diff
@@ -95,6 +96,8 @@ const prepareSiteHeaderCore = (data, lang) => {
   );
   // Extra classes and extra attributes.
   getExtraKnobs(data);
+  // Compliance.
+  getComplianceKnob(data);
```

Comment:
I covered this case, although here we fall in that "grey area" i mentioned earlier.
The `search_form.text_input.label` "generates" the hidden label of the text_field, the `search_form.text_input.id` sets an id on it and associates it with the aforementioned label.
Although we couldn't maybe say that the component violates a specific rule set by dg comm for this component, it would surely "fail" in an "accessibility" wise approach.

Reply:
No, sorry, my bad here, I had tested this after updating fields and had left it empty.
By default it's not empty, which is ok for me. My comment was for empty value by default on mandatory element which is not the case. It has a value by default and the details you have mentioned are to be displayed on modifying values, it's ok.